app/views/research_sessions/preview.html.erb

```diff
@@ -157,18 +136,112 @@
           <% end %>
         </p>
       <% end %>
-    </section>
-  <% end %>
-  <% if @research_session.incentives_enabled %>
-    <section>
-      <h3 class="subtitle-small">Incentives</h3>
-      <p>
-        We are offering
-        <%= edit_link_for(:incentive_value) do %><%= @research_session.incentive_text %><% end %>
-        for participation in this session.
-      </p>
-    </section>
-  <% end %>
+    <% end %>
+  </section>
+
+  <section>
+    <h3 class="subtitle-small" id="confidential">
+      Will anyone know what <%= i_or_they %> say in the discussion?
```

Comment:
There should be no question mark at the end

Reply:
@sampling Have to disagree. If it's a question it needs a question mark. We can remove the question mark if we rephrase it to not be a question.